tests/e2e/specs/in-game/one-offs/0_countering.spec.js

```diff
@@ -332,53 +332,44 @@ describe('Countering One-Offs P0 Perspective', () => {
     cy.setupGameAsP0();
   });
 
-  it('Can counter a three', () => {
-    cy.skipOnGameStateApi();
+  it('Increments turn when one-off fizzles', () => {
```

Comment:
This test case was named totally wrong compared to what it was actually testing since it wasn't actually playing a three that got countered; it was a five. I don't actually know what the heck happened there, but it seems to me that the rank itself won't matter much and that the key thing this test caught when we added it to the gamestate test scope was that the turn needs to increment when a one-off is countered. 

I've renamed and reworked the test case accordingly